src/test/resources/test-cases.csv

```diff
@@ -23491,4 +23491,5 @@ Bulawayo,-20.15,28.58333,Africa/Harare
 Bindura,-17.30192,31.33056,Africa/Harare
 Beitbridge,-22.21667,30,Africa/Harare
 Epworth,-17.89,31.1475,Africa/Harare
-Chitungwiza,-18.01274,31.07555,Africa/Harare
\ No newline at end of file
+Chitungwiza,-18.01274,31.07555,Africa/Harare
+Chicago IL,41.8369,-87.6847,America/Chicago
```

Comment:
I moved it from the hand entered test cases to the csv, but I can remove it if you think it's not necessary.